sdk/eventgrid/azure-eventgrid/azure/eventgrid/_operations/_patch.py

```diff
@@ -123,16 +148,23 @@ def publish_cloud_events(
         :rtype: None
         :raises ~azure.core.exceptions.HttpResponseError:
         """
-
+        # Do we want this in case they try to directly send in an event from SB or EH?
+        if isinstance(body, dict):
```

Comment:
an event from SB/EH shows up as a dict?

Reply:
If you receive a CloudEvent from SB/EH it would be in the form of a dictionary, and the customer would have to do CloudEvent.from_dict() before passing it in